### PR TITLE
Enforce the use of braces for conditional statements

### PR DESCRIPTION
**Pull request by Nihlus**
This PR enforces the use of braces for statements, disallowing their omitting for single-line expressions.

Discussions about this, and motivation thereof, can be viewed in #566.

### CODE REVIEW

**Comment by Nihlus** [comment]
@varon It seems we had a failure due to a temporary nuget outage. Could you restart the AppVeyor build?

**Comment by varon** [comment]
@Nihlus  - restarted

**Comment by varon** [comment]
Should have granted your permissions as well, I think.

**Comment by Nihlus** [comment]
@varon Perfect, thanks.

File: src/OpenTK/Platform/Factory.cs [hunk 1]
```diff
@@ -47,20 +47,37 @@ public Factory()
 
             // Create regular platform backend
             #if SDL2
```
Comment (Frassle) [inline src/OpenTK/Platform/Factory.cs]: Line up these hash ifs with the endifs

File: src/Generator.Converter/GLXmlParser.cs [hunk 2]
```diff
@@ -434,9 +449,13 @@ private string FunctionParameterType(XElement e)
             {
                 var words = ret.Split(" ".ToCharArray(), StringSplitOptions.RemoveEmptyEntries);
                 if (words[0] == "struct" || words[0] == "const")
-                    words[1] = group.Value;
+                {
+                    words[1] = @group.Value;
```
Comment (Frassle) [inline src/Generator.Converter/GLXmlParser.cs]: Inconsistent use of "@"

File: src/OpenTK/Platform/Windows/WinWindowInfo.cs [hunk 3]
```diff
@@ -74,8 +74,10 @@ public IntPtr DeviceContext
             get
             {
                 if (dc == IntPtr.Zero)
+                {
                     dc = Functions.GetDC(this.Handle);
-                    //dc = Functions.GetWindowDC(this.Handle);
```
Comment (Frassle) [inline src/OpenTK/Platform/Windows/WinWindowInfo.cs]: Comment should probably stay inside the if

**Comment by Nihlus** [comment]
@Frassle Done. I ended up removing the GetWindowDC call and method altogether - they were not used.

**Pull request merged**